### ISSUE TITLE
[Move/Copy/Link] Removing an original does not remove links

### ISSUE THREAD

**Issue by VWoeltjen** [nasa/openmct#173]
Removing an object that has links does not cause any links to be removed, or upgraded to originals.

Refer to UI docs for desired behavior; summary at the time of this writing is:
- Remove should be unavailable for originals
- Delete should be available for both links and originals. Delete should cause the original _and all links_ to disappear.

**Comment by VWoeltjen** [comment]
Think there's room for some flexibility in this approach - looking toward Unix filesystems for inspiration, hard links are destroyed only after the last removal, whereas symlinks just break when the original is removed. Think it might be fine to let links simply be replaced with "missing object" markers, since that will be simplest to implement, and in some cases it may be useful to know that something _was_ here (e.g. in a Layout), but got deleted elsewhere.

**Comment by larkin** [comment]
I think this will be resolved when "delete" is added: you "remove" a link, you "delete" an original.  Removing an original is not an option, nor can one "delete" a link.  Deleting an original would break all links-- where as removing a link would not do the same.

**Comment by larkin** [comment]
consider this a wontfix at the present time.
